### DISCUSSION TITLE
I need to parse the json response of GET /tickets/[ticket id] in the API

### DISCUSSION THREAD

**Discussion by rodrigolfh** [django-helpdesk/django-helpdesk#1223] [Q&A] [answered]
Hi!
The app we're using in my work needs to receive a json response with no escape characters in the 'description' field of the returned ticket, or even better i could replace all the single escape characters with a double escape character.

I managed to find the view method that manages the corresponding ticket and I also managed to alter the description to any string.

But when it comes to use .replace() to replace those \'s I have no success.

views/api.py:

class TicketViewSet(viewsets.ModelViewSet):
...
    def get_object(self):
        ticket = super().get_object()
        ticket.set_custom_field_values()
        #ticket.description = ticket.description.replace("\\", "")
        #ticket.description = json.loads(ticket.description.replace("\\", ""))
        #ticket.description = ticket.description.replace(r"\", "")
        #ticket.description = re.sub(r"\\", "", ticket.description)
        #ticket.description = ticket.description.replace("'","")
        #ticket.description = 'hola' # < this actually works

        return ticket

You can see my tries.

Any clue?

**Comment by uhurusurfa** [comment]
Have you tried something like this:
https://stackoverflow.com/questions/19176024/how-to-escape-special-characters-in-building-a-json-string/79203056#79203056

**Comment by rodrigolfh** [comment, marked as answer]
Finally managed to do it using replace("/n", //n"), that is putting the backslash along with the n letter (or r in some cases)
